opensearch-cluster-cdk/lib/opensearch-config/node-config.ts

```diff
@@ -0,0 +1,43 @@
+/* Copyright OpenSearch Contributors
+SPDX-License-Identifier: Apache-2.0
+
+The OpenSearch Contributors require contributions made to
+this file be licensed under the Apache-2.0 license or a
+compatible open source license. */
+
+export const nodeConfig = new Map<string, object>();
+
+nodeConfig.set('master', {
+  'node.name': 'master-node',
+  'node.master': true,
+  'node.data': false,
+  'node.ingest': false,
+});
+
+nodeConfig.set('data', {
+  'node.name': 'data-node',
+  'node.master': false,
+  'node.data': true,
+  'node.ingest': true,
+});
+
+nodeConfig.set('seed-master', {
+  'node.name': 'seed',
+  'node.master': true,
+  'node.data': false,
+  'node.ingest': false,
+});
+
+nodeConfig.set('seed-data', {
```

Comment:
Not related with this line. Can we support creating ML nodes?

Reply:
Ack! @ylwu-amzn